stock-filters/ChirpVillageBuilder.py

```diff
@@ -1,35 +1,21 @@
 from pymclevel import alphaMaterials, MCSchematic, MCLevel, BoundingBox
-from BasicBuilding import BasicBuilding 
+from BasicBuilding import BasicBuilding
 from Surface import Surface
 import utilityFunctions
+from random import randrange, uniform
+from ChirpVillage import BuildPlaneFinder, Builder
 import BlockUtils
 import os
 
 inputs = (
     ('Build Chirp Village', 'label'),
-    ('Material1', alphaMaterials.StoneBricks),
     ('Creator: Chirp Nets', 'label'),
 )
-default_path = '/stock-filters/ChirpVillage/Buildings/'
-files = [
-    'ghast.schematic',
-]
 
 
 def perform(level, box, options):
-    surface = Surface(box.minx, box.minz, box.maxx, box.maxz)
+    surface = Surface(level, box)
     BlockUtils.calculate_biomes_on_surface(level, surface)
-    BasicBuilding(level, box, surface)
```

Comment:
Did we want to keep this for testing purposes?

Reply:
The declaration for the building changed, I removed it because it requires the yard stuff to work with the surface